frontend/src/constants/index.ts

```diff
@@ -10,8 +10,7 @@ window.config = window.config || {
 };
 
 export const ENTER_KEY = 13;
-export const API_URL = window.config.API_URL;
+export const API_URL =window.config.API_URL;
```

Comment:
unrelated change. could you please remove this?

Reply:
yeah will remove it.